http_prompt/commandio.py

```diff
@@ -0,0 +1,33 @@
+import sys
+import six
+
+from .printer import Printer
+from .utils import strip_color_codes
+
+
+class CommandIO():
```

Comment:
We also need to handle the append file output redirection option `> post >> /tmp/out.txt` which I don't think is supported by the httpie. We could also support error output redirections in the future. I think it's good idea to have some abstraction layer on top of the common interface which connects all output redirection features together. In my optinion, the execution.py file should be minimal as possible and what's possible should be put into separate files to handle custom logic... Another example is the httpie request/response logic which I've put also into separate file..